pages/utils/sum.ts

```diff
@@ -0,0 +1,8 @@
+export function sum(a: number, b: number, format: "html" | "text" = "html") {
```

Comment:
Kindly create a typing that contains the format value as shown below

type Format: "html" | "text" 

And use the format type in the format parameter as a type

Reply:
Good catch! Will do.